content/manager/broker-security.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: bt_wiki
```

Comment:
please remove all references to 3.3. This is the 3.3 docs branch.. :)
Not having versions inside the docs will make them more manageable.